storehaus-elasticsearch/src/main/scala/com/twitter/storehaus/elasticsearch/ElasticSearchStringStore.scala

```diff
@@ -70,10 +66,10 @@ class ElasticSearchStringStore(private val index: String,
         val response = request.execute().actionGet()
 
         response.iterator().asScala
-          .filter(r=>Option(r.getResponse)!=None)
+          .filter(r => Option(r.getResponse).isDefined)
```

Comment:
Is there a reason to prefer `Iterator.empty` / `Iterator.single` over `None` / `Some`?